bin/Cargo.toml

```diff
@@ -73,16 +73,16 @@ path = "src/named.rs"
 [dependencies]
 clap = { version = "3.0", default-features = false, features = ["std", "cargo"] }
 futures = { version = "0.3.5", default-features = false, features = ["std"] }
-log = "0.4"
 rustls = { version = "0.20", optional = true }
 time = "0.3"
+tracing = "0.1"
+tracing-subscriber = "0.2"
```

Comment:
should tracing-subscriber be a dev-dependency? it is roughly equivalent in usage to env-logger, right?